components/CatList/CatList.js

```diff
@@ -0,0 +1,41 @@
+import { cats } from "../../data/catdata";
+import CatCard from "../CatCard/CatCard.js";
+import { styled } from "styled-components";
+
+export default function CatList() {
+  return (
+    <>
+      <StyledHeading>Your Cats🐾</StyledHeading>
+      <StyledList>
+        {cats &&
```

Comment:
could you explain why you use the && operator here? Maybe i understood the operator wrong

Reply:
first i have to check if the array is filled at all, if yes then i can map the list and display each cat as a card

Reply:
ahhhh thank you !

Reply:
smart